app/gui/src/components/TabView.vue

```diff
@@ -147,4 +163,30 @@ const onSignOut = () => {
   min-height: 0;
   display: flex;
 }
+
+.closeButton {
+  --button-padding: 0px;
+  --icon-width: 12px;
+  --icon-height: 12px;
+
+  &:hover {
+    background-color: rgba(239, 68, 68, 0.8);
+  }
+}
+
+.App.onMacOs .closeButton {
+  transition: color 0.1s;
+  color: rgba(0, 0, 0, 0);
+  &:not(:hover) {
+    background-color: rgba(0, 0, 0, 0.3);
+  }
+  &:hover {
+    color: rgba(0, 0, 0, 0.9);
+  }
+}
+
+.SelectableTab span {
+  /* Trim the text box, so the span will align with icons properly*/
+  text-box: trim-both cap alphabetic;
```

Comment:
Avoid using text-box since it has very limited availability across the browsers: https://developer.mozilla.org/en-US/docs/Web/CSS/text-box-trim

(even our electron doesn't support it)